packages/compress-stringify/typescript/src/pipeline-worker-url.ts

```diff
@@ -1,10 +1,21 @@
+// @ts-ignore: TS2305: Module '"itk-wasm"' has no exported member 'getPipelineWorkerUrl'.
+import { getPipelineWorkerUrl as itkWasmGetPipelineWorkerUrl } from 'itk-wasm'
 import packageJson from '../package.json'
-let pipelineWorkerUrl: string | URL | null = `https://cdn.jsdelivr.net/npm/itk-compress-stringify@${packageJson.version}/dist/web-workers/pipeline.worker.js`
+
+let pipelineWorkerUrl: string | URL | null | undefined
+let defaultPipelineWorkerUrl: string | URL | null = `https://cdn.jsdelivr.net/npm/itk-compress-stringify@${packageJson.version}/dist/web-workers/pipeline.worker.js`
```

Comment:
`const` if not going to modify? Also, doesn't need to have the extra union types.